tools/tsp-client/src/fs.ts

```diff
@@ -1,8 +1,8 @@
-import { mkdir, rm, writeFile, stat, readFile, access } from "node:fs/promises";
-import { FileTreeResult } from "./fileTree.js";
+import { mkdir, rm, stat, readFile, access } from "node:fs/promises";
```

Comment:
does it work if you import from "fs/promises"? without `node:` just curious, as I'm not used to this notation.

Reply:
It does seem to work without it. I'm not sure why this was added for me. I could look into removing it in the future if it's better. Let me know! I'll merge this PR in without that change though